servicetalk-concurrent-api/src/main/java/io/servicetalk/concurrent/api/TaskBasedAsyncPublisherOperator.java

```diff
@@ -246,8 +264,12 @@ private void offerSignal(Object signal) {
             }
 
             try {
-                LOGGER.debug("delivering Subscriber signals on {}", executor);
-                executor.execute(this::deliverSignals);
+                if (offload.getAsBoolean()) {
+                    LOGGER.trace("delivering Subscriber signals on {}", executor);
```

Comment:
remove debug/development logs? operators allow for instrumentation/visibility if required.

Reply:
I have fixed a few cases where `debug` was used rather than `trace` but am reluctant to remove them entirely. They do provide more information than the operators can access.

Reply:
in cases where we can add instrumentation at higher layers lets take advantage of that as we have more flexibility (customize how info is captured ... logging, format, metrics, etc.). In this case we have visibility into the executor but in most cases you can derive that info at higher layers from the currentThread() IIUC.